hcloud/actions/client.py

```diff
@@ -36,6 +50,18 @@ def get_list(self,
                  per_page=None,  # type: Optional[int]
                  ):
         # type: (...) -> PageResults[List[BoundAction]]
+        """Get a list of actions from this account
+
+        :param status: List[str] (optional)
+               Response will have only actions with specified statuses. See `our documentation <https://docs.hetzner.cloud/#actions-list-all-actions>`_  for all available values.
+        :param sort: List[str] (optional)
+               Specify how the results are sorted. See `our documentation <https://docs.hetzner.cloud/#actions-list-all-actions>`_  for all available values.
+        :param page: int (optional)
+               Specifies the page to fetch
```

Comment:
```suggestion
               Page number to fetch
```
Drop the word "Specify" everywhere

Reply:
I've got the same naming like in the API Docs. I find we should equal with the docs.